# fix early build error. The calling function expects to get something like
# "i686", but gets CPU vendor's free form processor name ("Intel Core Duo" 
# or something) 

--- a/rosetta++/tools/build/setup_platforms.py	2008-04-17 22:26:55.000000000 +0200
+++ b/rosetta++/tools/build/setup_platforms.py	2008-12-04 18:33:21.000000000 +0100
@@ -270,7 +270,7 @@
     """
     # Note that these are very different values, and need processing
     if globals().has_key("platform"):
-        return platform.processor() or _uname()[4]
+        return platform.machine() or _uname()[4]
     else:
         return _uname()[4]
 
